CONTRIBUTING.md

```diff
@@ -28,6 +28,25 @@ There are also a few additional workflows that can be triggered manually:
 
 Members of the repository can trigger these workflows in the "Actions" tab.
 
+### Running the CI locally
+
+You can also run GitHub Actions locally with [act](https://github.com/nektos/act).
+
+List the workflows with `act --list`. To simulate a push event, run `act push`. Some workflows might work on GitHub Actions but not on act, in which case you could try running specific workflows with, for example, `act -j 'build'`.
```

Comment:
In particular, I could run the first tutorial, but not more. It was as if the second solver was not starting.

Reply:
I added a step to install CMake for the Quickstart tutorial. Thanks for spotting and reporting this.